### PR TITLE
ModuleBuilder - removing duplicated language strings  [Language]

### PR DESCRIPTION
**Pull request by horus68**
Removing duplicated languages strings on same array
string  'LBL_CONFIRM_FIELD_DELETE' = Making actual multinline into a single line for translations

### CODE REVIEW

File: modules/ModuleBuilder/language/en_us.lang.php [hunk 1]
```diff
@@ -320,7 +320,6 @@
             'layoutsHelp' => 'Select a <b>Layout to edit</b>.<br/<br/>To change the layout that contains data fields for entering data, click <b>Edit View</b>.<br/><br/>To change the layout that displays the data entered into the fields in the Edit View, click <b>Detail View</b>.<br/><br/>To change the columns which appear in the default list, click <b>List View</b>.<br/><br/>To change the Basic and Advanced search form layouts, click <b>Search</b>.',
             'subpanelHelp' => 'Select a <b>Subpanel</b> to edit.',
             'searchHelp' => 'Select a <b>Search</b> layout to edit.',
-            'labelsBtn' => 'Edit the <b>Labels</b> to display for values in this module.',
```
Comment (samus-aran) [inline modules/ModuleBuilder/language/en_us.lang.php]: Why delete this label - this is the label that is used currently - would it not be best to delete the other duplicate?

Comment (samus-aran) [inline modules/ModuleBuilder/language/en_us.lang.php]: Should this one be deleted?

**Comment by horus68** [comment]
Agree with the switch. I was not aware of what line was used

File: modules/ModuleBuilder/language/en_us.lang.php (hunk 1 above)
Comment (horus68) [inline modules/ModuleBuilder/language/en_us.lang.php]: ok for me. changed the file

**Pull request closed without merging**